tests/unit/pudl_archiver_test.py

```diff
@@ -55,6 +55,14 @@ async def test_archive_datasets(
     tmp_path,
 ):
     """Test that archive datasets creates run_summary file."""
+    mocker.patch.dict(
```

Comment:
I accidentally ran this test without the corresponding environment variables set, and figured it best to mock them here since it's a unit test (vs. an integration test which should probably expect those env vars).